src/server/db/user.ts

```diff
@@ -45,15 +46,22 @@ export const insertUserWithAddress = async (address: string) => {
 
 export const selectMatchPlayedByUser = async (
   userId: string,
+  days?: number,
```

Comment:
Param name is ambiguous. Provide a better name so one can understand what it means without having to look at the code